detection_rules/schemas/__init__.py

```diff
@@ -17,15 +17,17 @@
     "downgrade",
     "CurrentSchema",
     "validate_rta_mapping",
+    "schema_map",
     "TomlMetadata",
 )
 
-all_schemas = [
-    ApiSchema78,
-    ApiSchema79,
-    ApiSchema710,
-    ApiSchema711,
-]
+schema_map = {
+    '7.8': ApiSchema78,
+    '7.9': ApiSchema79,
+    '7.10': ApiSchema710,
+    '7.11': ApiSchema711
+}
```

Comment:
can you actually leave the list as it was? The target version is a class property, so you can recover it dynamically. can you a new variable for the `available_versions = [cls.STACK_VERSION for cls in all_schemas]`

I lnow that python dictionary ordering is now deterministic and all that but I'm not ready to rely on it yet